zebra-consensus/src/block.rs

```diff
@@ -99,6 +99,10 @@ where
         async move {
             let hash = block.hash();
 
+            // The height is already included in the ChainVerifier span
+            let span = tracing::debug_span!("BlockVerifier.call", ?hash);
```

Comment:
Apply the same suggestion here
```suggestion
            let span = tracing::debug_span!("BlockVerifier::call", ?hash);
```